.github/workflows/CI.yml

```diff
@@ -14,41 +14,79 @@
 # limitations under the License.
 
 ---
-name: Code Formatting
+name: Daffodil CI
```

Comment:
Change this to be `Daffodil VS Code CI`

Reply:
I would actually just suggest `CI`. We did this for Daffodil because in some places GitHub displays jobs like `Workflow Name / Job Name`. Sometimes GitHub doesn't proide much space for this, so it cuts off the job name, which is the part that actually matters. So just changing to "CI" ensures the job is always visible.